conjure_python_client/_http/requests_client.py

```diff
@@ -29,20 +29,20 @@
 
 T = TypeVar("T")
 # https://testssl.sh/openssl-rfc.mappping.html
+# https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28recommended.29
 CIPHERS = (
-    "ECDHE-RSA-AES256-SHA384:"
-    "ECDHE-RSA-AES128-SHA256:"
-    "ECDH-RSA-AES256-SHA384:"
-    "ECDH-RSA-AES128-SHA256:"
-    "AES128-SHA256:"
-    "AES256-SHA256:"
-    "ECDHE-RSA-AES256-SHA:"
-    "ECDHE-RSA-AES128-SHA:"
-    "ECDH-RSA-AES256-SHA:"
-    "ECDH-RSA-AES128-SHA256:"
-    "AES256-SHA:"
-    "AES128-SHA:"
-    "TLS_FALLBACK_SCSV"
+    # don't include the tls 1.3 ones
```

Comment:
Why are we excluding the TLS1.3 ciphers?

Reply:
My understanding from @bavardage is we cannot use TLSv1.3 yet

Reply:
there's some discussion in https://github.com/urllib3/urllib3/issues/1550, but this is based on a recommendation from @iambenjackson

Reply:
https://docs.python.org/3/library/ssl.html#ssl.SSLContext.set_ciphers impies that in tls 1.3 mode we already just get tls 1.3 ciphers?

Reply:
right we can't force tls 1.3 only

Reply:
hmm ok, if we can't opt out of the 1.3 ciphers we might as well put the them in the list so its explicit what cipher we expect to see. In practice it won't really matter since WC will prefer other cipher suites anyways

Reply:
fine to give it a whirl